website/app.js

```diff
@@ -28,7 +28,7 @@ export function createExecTimeColumns(data) {
     ...data.map(d => {
       const benchmark = d.benchmark[name];
       const meanValue = benchmark ? benchmark.mean : 0;
-      return meanValue || 0;
+      return meanValue == 0 ? null : meanValue;
```

Comment:
FYI,
```js
let a = 0 || null; // a is 0
```
JS treats 0 as falsy, so no need for ternary operator, simply
```js
return meanValue || null;
```
would work